src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let result = 40 * days;
+
+  if (days > 6) {
+    result -= 50;
+  } else if (days > 2) {
+    result -= 20;
+  }
```

Comment:
You should use "switch" operator, because if you want to support your code by adding more conditions. For example next step you need to add a condition discount if you have 10 days your discount is -100$. Your code will look like this: if else {} if else {}if else {}if else {}if else {}... and your first part of the code always needs to change if => else if...